qrisk/stats.py

```diff
@@ -353,15 +431,22 @@ def sortino_ratio(returns, required_return=0, period=DAILY):
         Annualized Sortino ratio.
 
     """
-    try:
-        ann_factor = ANNUALIZATION_FACTORS[period]
-    except KeyError:
-        raise ValueError(
-            "period cannot be: '{}'."
-            " Must be '{}', '{}', or '{}'".format(
-                period, DAILY, WEEKLY, MONTHLY
+
+    if len(returns) < 2:
+        return np.nan
+
+    if annualization is None:
```

Comment:
it is just an approximation, there are other years that have less trading days, for example when hurricane sandy closed the market.